modules/installation-azure-service-principal.adoc

```diff
@@ -51,6 +48,7 @@ $ az cloud set -n <environment_name>
 ----
 $ az cloud update --profile 2019-03-01-hybrid
 ----
+// Can we update this profile name?
 
 endif::ash[]
 
```

Comment:
@MayXuQQ 
Is logging in via the web console an additional step? Or an alternative to logging in to the Azure CLI?

Reply:
login with the azure cli

Reply:
@MayXuQQ Please see above question. Does the user have to log in via the web console after logging in with the CLI?

Reply:
after login with CLI, need not login via the web console.